agent/build.gradle

```diff
@@ -40,6 +40,7 @@ dependencies {
 
     testImplementation 'junit:junit:4.12'
     testImplementation 'org.assertj:assertj-core:3.8.0'
+    testImplementation 'org.mockito:mockito-core:2.+'
```

Comment:
Please replace with a fixed version as variable versions make the build non-reproducible. Just ran into a major issue when updating mockito in TS today ;-)